.github/workflows/verify_failure.yml

```
@@ -0,0 +1,35 @@
+      - name: Verify Assert
+        run: |
+          cd test 
+          if ./build/Example; then
+            echo "ERROR: Test should have failed but passed"
+            exit 1
+          else
+            echo "Test failed as expected"
+            exit 0
+          fi
+     
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)